rasa/test.py

```diff
@@ -137,6 +137,7 @@ def test_core(
             "Rasa model and provide it via the '--model' argument."
         )
 
+    # TODO: test for classifier
```

Comment:
better comment would have been: `test for real classifier` as we currently get the `KeywordClassifier`